op-node/rollup/derive/channel.go

```diff
@@ -45,11 +47,12 @@ type Channel struct {
 	highestL1InclusionBlock eth.L1BlockRef
 }
 
-func NewChannel(id ChannelID, openBlock eth.L1BlockRef) *Channel {
+func NewChannel(id ChannelID, openBlock eth.L1BlockRef, holocene bool) *Channel {
 	return &Channel{
 		id:        id,
 		inputs:    make(map[uint64]Frame),
 		openBlock: openBlock,
+		holocene:  holocene,
```

Comment:
I wonder if it would be better to name this based on what it does rather than the hard fork.  ie `requireInOrder`.  We'd still pass `IsHolocene` into it but to me it makes the code easier to read.

Reply:
renamed